source/features/deep-reblame.css

```diff
@@ -0,0 +1,7 @@
+a:not([href]).rgh-deep-blame .octicon-versions {
```

Comment:
⬆️

Reply:
I didnt fix it since 

> @fregante looks like if I add an href github replaces the ref with a direct link to the same line. breaking the entire feature

try it you will see

Reply:
> @fregante looks like if I add an href github replaces the ref with a direct link to the same line. breaking the entire feature

It's us: `link-to-prior-blame-line`

Reply:
ahh so we need to run that after